ci/gpu/build.sh

```diff
@@ -49,7 +64,7 @@ logger "Build libcuml..."
 mkdir -p $WORKSPACE/cuML/build
 cd $WORKSPACE/cuML/build
 logger "Run cmake libcuml..."
-cmake -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX -DCMAKE_CXX11_ABI=ON ..
+cmake -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX -DCMAKE_CXX11_ABI=ON -DGPU_ARCHS="$GPU_ARCH" ..
```

Comment:
Is the `DGPU_ARCHS` flag needed for libcuml's cmake?

Reply:
It is optional, but reduces the time of compilation significantly. Since the package building GPUCI scripts already check building for all architectures, building for more than one GPU_ARCH in GPU tester doesn't add any coverage unless we ran the tests in all the gpu architectures.